tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py

```diff
@@ -309,7 +309,8 @@ def summarize_bug_info(projects):
                      "resolution": bug["resolution"]})
     bugs_list = sorted(list(bugs.items()))
     for bug_id, bug in bugs_list:
-        bug["changes"].sort(key=lambda x: x["commit"]["change"]["number"])
+        if bug["changes"].sort(key=lambda x: x["commit"]) != None:
```

Comment:
In what case does the list.sort() return something else than None?

Reply:
faux pas, you got right, that sort is basically not needed here.